static/src/javascripts/lib/dynamic-import-init.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import dynamicImportPolyfill from 'dynamic-import-polyfill';
+
+// Provides an import function to use for dynamic imports. **Only works on
+// browsers that support modules.**
+const initialiseDynamicImport = () => {
+    try {
+        /* eslint-disable no-new-func */
+        window.guardian.functions.import = new Function(
+            'url',
+            `return import(url)`,
```

Comment:
Right very interesting! Avoid that eval 👍 

```suggestion
		// Resolves to: import = (url) => import(url);
        /* eslint-disable no-new-func */
        window.guardian.functions.import = new Function(
            'url',
            `return import(url)`,
```

Reply:
Shamelessly copied from Nic's PR.